gcloud-java-bigquery/src/main/java/com/google/cloud/bigquery/TableInfo.java

```diff
@@ -63,6 +64,7 @@ public Table apply(TableInfo tableInfo) {
   private final Long creationTime;
   private final Long expirationTime;
   private final Long lastModifiedTime;
+  private final TimePartitioning timePartitioning;
```

Comment:
Sure thing! I'll cherry pick your commit and open a new PR, I'll /cc you as soon as that is ready so you can have a look and let me know if it suits your needs, how does this sound?